Automattic/delphin
# Display Google Translate attribution

Fixes #208.

<img width="1105" alt="screen shot 2016-06-21 at 6 05 07 pm" src="https://cloud.githubusercontent.com/assets/1130674/16248027/c569b6e8-37da-11e6-819a-54a99ec93364.png">

^ misérable?

**Testing**
- Visit http://delphin.localhost:1337/search?q=dogs
- Click on 'dogs'
- Assert that you do not see a 'Powered by Google Translate' image.
- Visit http://delphin.localhost:1337/fr/search?q=dogs
- Click on 'dogs'
- Assert that you do see a 'Powered By Google Translate' image.
